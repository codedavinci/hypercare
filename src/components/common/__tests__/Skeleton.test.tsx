import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Skeleton from "../Skeleton";

describe("Skeleton Component", () => {
  it("should render correctly and match snapshot", () => {
    const { asFragment } = render(<Skeleton />);
    expect(asFragment()).toMatchSnapshot();
  });
});
