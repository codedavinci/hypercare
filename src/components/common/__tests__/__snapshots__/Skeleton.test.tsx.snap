// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton Component should render correctly and match snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col gap-4"
  >
    <div
      class="flex gap-4 items-center"
    >
      <div
        class="skeleton w-16 h-16 rounded-full shrink-0"
      />
      <div
        class="flex flex-col gap-4"
      >
        <div
          class="skeleton h-4 w-20"
        />
        <div
          class="skeleton h-4 w-28"
        />
      </div>
    </div>
    <div
      class="skeleton h-32 w-full"
    />
  </div>
</DocumentFragment>
`;
